### ISSUE TITLE
Can't view details of some tasks from Nextcloud

### ISSUE THREAD

**Issue by melianmiko** [mrvladus/Errands#137]
When clicking on some tasks, got this error and details panel don't appear:

```
[DEBUG] Details: Update info
Traceback (most recent call last):
  File "/usr/share/errands/errands/widgets/task.py", line 296, in on_details_clicked
    self.details.update_info(self)
  File "/usr/share/errands/errands/widgets/details.py", line 278, in update_info
    self.start_datetime.set_datetime(self.parent.get_prop("start_date"))
  File "/usr/share/errands/errands/widgets/components.py", line 194, in set_datetime
    self.hour.set_value(int(dt[9:11]))
                        ^^^^^^^^^^^^^
ValueError: invalid literal for int() with base 10: ''
```

This task is created by TasksOrg app on Android, here's how it looks like in browser network inspect:

```
<cal:calendar-data>BEGIN:VCALENDAR
VERSION:2.0
PRODID:+//IDN tasks.org//android-130605//EN
BEGIN:VTODO
DTSTAMP:20231229T051546Z
UID:2754374180015068324
CREATED:20231229T051509Z
LAST-MODIFIED:20231229T051540Z
SUMMARY:<<Super-secret task summary hidden>>
PRIORITY:5
X-APPLE-SORT-ORDER:725173819
DUE;VALUE=DATE:20240131
DTSTART;VALUE=DATE:20240114
BEGIN:VALARM
TRIGGER;RELATED=END:PT0S
ACTION:DISPLAY
DESCRIPTION:Default Tasks.org description
END:VALARM
END:VTODO
END:VCALENDAR
</cal:calendar-data>...
```

**Comment by mrvladus** [comment]
Found the issue, fixed now in 08076d4cd8d09e6a7e570054d5199dc2d24928f5. Thanks!

**Comment by melianmiko** [comment]
Yes, now it works. 

Thank you for your wonderful application)
